jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java

```diff
@@ -150,35 +150,52 @@ public static Map<String,Object> formErrorCode() {
         return errorCode;
     }
 
-	public static void omitOpsAlert() {
-		List<PermissionVH> permissionIssue=new ArrayList<>();
-		for(Map.Entry<String,List<ErrorVH>> entry:errorMap.entrySet())
-		{
-			List<ErrorVH> errorVHList=entry.getValue();
-			for(ErrorVH errorVH:entry.getValue())
-			{
-				if(errorVH.getType().equals("phd")&&(errorVH.getException().contains("AccessDeniedException")||errorVH.getException().contains("SubscriptionRequiredException")))
-				{
-					PermissionVH permissionVH=new PermissionVH();
-					log.info("Omit exception :{}",errorVH.getException());
-					permissionVH.setAccountNumber(entry.getKey());
-					permissionVH.setErrorVH(errorVH);
-					permissionIssue.add(permissionVH);
-					errorVHList.remove(errorVH);
-				}
-				if(((errorVH.getType().equals("kms")||errorVH.getType().equals("s3"))&&errorVH.getException().contains("AccessDeniedException"))||(errorVH.getType().equals("checks")&&errorVH.getException().contains("AWSSupportException")))
-				{
-					PermissionVH permissionVH=new PermissionVH();
-					log.info("Omit exception :{}",errorVH.getException());
-					permissionVH.setAccountNumber(entry.getKey());
-					permissionVH.setErrorVH(errorVH);
-					permissionIssue.add(permissionVH);
-					errorVHList.remove(errorVH);
-				}
-			}
-			if(errorVHList.isEmpty()){errorMap.remove(entry.getKey());}
-		}
-		//commenting to avoid too many requests
-//		NotificationPermissionUtils.triggerNotificationsForPermissionDenied(permissionIssue,"AWS");
-	}
+    public static void omitOpsAlert() {
+        List<PermissionVH> permissionIssue = new ArrayList<>();
+        for (Map.Entry<String, List<ErrorVH>> entry : errorMap.entrySet()) {
+            PermissionVH permissionVH = new PermissionVH();
+            permissionVH.setAccountNumber(entry.getKey());
```

Comment:
cannot move it outside since account number is coming -> entry.getKey() -> which is coming from ->  for (Map.Entry<String, List<ErrorVH>> entry : errorMap.entrySet())

Reply:
I'm not talking to move it outside, I'm talking opposite. You have `if (!assetPermissionMapping.isEmpty()) {` down below. You can move this object initialization right there, inside that `if` section.

Reply:
Addressed in commit